modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -1183,7 +1183,13 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       .result();
   }
 
-  static publicKeyFromCommonKeychain(commonKeychain: string) {
+  /**
+   * util function that checks that a commonKeychain is valid and can ultimately resolve to a valid public key
+   * @param commonKeychain - a user uploaded commonKeychain string
+   * @throws if the commonKeychain is invalid length or invalid format
+   */
+
+  static getUncompressedPublicKeyFromCommonKeychain(commonKeychain: string) {
```

Comment:
This naming is little clearer
```suggestion
  static validateCommonKeychainPublicKey(commonKeychain: string) {
```